docs/testrunner-toolkit/faqs.md

```diff
@@ -1,25 +1,18 @@
 ---
 id: faqs
-title: Testrunner FAQs
+title: Sauce Control FAQs
 sidebar_label: FAQs
 ---
 
-Below are list of frequently asked questions regarding some of the business and technical use cases surrounding the Testrunner Toolkit.
+Below are list of frequently asked questions regarding some of the business and technical use cases surrounding Sauce Control.
 
-## How Does Testrunner Toolkit Work?
-Native JavaScript testing is achieved through the combination of Sauce Labs, Jest, and the
-JavaScript framework of your choice. In the current beta, the toolkit supports:
+## How Does Sauce Control Work?
 
-* [Puppeteer](https://github.com/puppeteer/puppeteer)
-* [Playwright](https://github.com/microsoft/playwright)
-* [TestCafe](https://github.com/DevExpress/testcafe)
-* [Cypress](https://github.com/cypress-io/cypress)
+Native testing is achieved through the combination of Sauce Labs, Jest, and the framework of your choice, giving you the power and expressiveness of different test frameworks with the dashboards, infrastructure, and analytics of [Sauce Labs](https://saucelabs.com/).
```

Comment:
`Jest` is only really used as part of puppeteer now.
Cypress uses `mocha`, playwright uses its own flavor `playwright-test`, testcafe also has its own.
So I'd suggest simply to skip mentioning Jest in this context :D